loculus_values/environment_specific_values/staging.yaml

```diff
@@ -34,3 +34,4 @@ secrets:
       password: AgCz5eedh7ZgfqRdeqMmlm/LK5xqLAd1ztJqE0eCg7d1y2W28C+Co7UelOaNPXMr6teADmbAAtk8JYTmaIMi8RRylCxFgiXZm7mimETk9V11KF9kNeOj0krT+KE8vI+GM5gq/jVW/AygwVflwe5UbjQYWG3rWVS2e5Uj/EagW3D73moxw2hzA7vNNGiKa/1OVfW6rm9RcwgBWBuz+aaTCwJ0CCO2iLLvubHU8dm8CLTnh/OjKtETdwNm0sBHHgNazvVqaXoNF5oqy0ItwBbhXJZQOB3BP5ITosreQYRjIAyyyhh7KBd2vJPPh8577nOswoLMyfEMUD3aNArDguLhM5XOaCaUE5GupzZi5UBDuaunMEgTSNs6Y+xxQNYP426xydNJgMJGBx2f1VNYOCH6vmmIXrVJETRAR41UFaqViZOGlM97iVhJqgY/ySRi8tDlYv3n7YrTnK1Tse44wEfN5aEbQKOl88qdcOiLFxhtxYhWKyhLlJvOxxaB4yXKNMeTg9TkfbUcroC9gUscB1RRMfifTxLlCuUt4GVxDO+1p7pTNaoPUOaRqss4r0uB2V7PTmqyzHu/qrMhU903V4oioxqYQVkPgA0xZLFEUTavZHBHiXU4BDSa8Ks4g+GJiwr4FTdFJHSBAKocQO1036sLvLcCGYBV0EcBUEjKrAyPyte2Jc2296ipaSHPFJaKG2R56h4ZP/fFYHjrLRZQ
       port: AgCVKaOX9s+R408WcJowEPaLZqz85/3nu6+q9DBExfFf/WArGvtPzoQIGX0n1ySgw+5skKMbcwhcc4jkbIrvKnphE0VexufFUyM3jFl9NuV+9dR2fSrEyUMgd3l5G5O0vyayL0ERgWi7DMcvpcbN7gfUrfzdWhNa7vhmAtWIZe+Hvff5lp2WKx8fVf4pPzDee0dfstCxQPdrF+N7/B6nN8k1kKiyDgxgx5YDSNpxSVWrpL2a3TfIlaubJTYplg62m0WSwPVcCNBnQaS7m75DwYLW/vTyAO3krNgXYqwaafPmkpZJmc6qT8RZE1e1sVizL7thfcJiZrOvZsxED9XYkEca6nQjyuV6EWubdymbxfhbmQy1AXsVXmCcbOrKGPv1GDui+EUw3dUiSD68+1VhUmy7EH/yYI4M2dZRtemJQPTiJhW8NfPqtZutkMdJGE5/dtjGTJ8OLeMiCz4TYyEqYwEFiUr8mfrLibdSdW6O9NAUP5niFx/5lNjNA19Xpu/8fJI0GUMY8s+UyPQC4W0xkJmof8zuZtq/n0r9PNSXIJulFQ6OTSaxMYqeHSG0xRVMoVs1/TPipjjyH8ssMtNK52ahWv4xNFEvoA6phLlXBAHBj4rHLVFDofnF9P2FZDQSmsevtAtc58wHvvI4Xe1skiKr4orUfZWVUaSuyQcHqub95XxwYQaMi7zGKfOZBcpiPvL8xFll
       username: AgAuZZ8Vs8bvPb1xqVEVNhQ5GpAL9f9gPfVNMCX1tT3NEXAPZn1/ujf6dl/ltaIeaO/DITWkC6YmODCcAvx5ABiwhYnDO7rY6V+UdIqTjQuPuoIf80L+kipf3ZTID/0nlS9yFMnSu++XdQqXJUGbPuOMyg9gMollWPyJJzTmQZQLYsXlvFrs0pifZzCbYwpRwyh3aCgMh6gZMdOlIGRR1n9xxY8oX2glx8+XdN3hoK3AEHf1E40/HZjLBGr3me+0gKvjw0kC3rQFJpHL917ODGB/3fJKT/rFF3CKMQLMkRV7ov6OUo+nKecDDEn3t8lDfWGPl3efKpZqyX5zIVz82r9q7qXWR3IluExZvp4gHncciuESO+LYONtS3faKppz3YBqEo14dyJtAVuph9evgfgl9yhtotfNUeu8DcXq9WrYbFhqnoSRrSOrslnAOgW+h0MJhbqoNTolaSqm/u/W45d9fKbPG53y9g8og/QnNF73sJLhq0P3NPMiZUUHhch5zcD3BLwr4RNd22ql0s0mxPtuRtXWWHgsQD5SZnCHSc5qfdcoDk2EkSyvNy1MzGhjt4itQUwawU98YvzU3jtiEnrkCkE3r0U5iyU+esajCam3HYorwSor9yjAIP1/P9ZSw7jKushqgh1FDXmZ0WXgvfXjbQKu5Ik63nEvJOGQ/gyuGc6QFR6Gh2YMZv85KEjvxbzydBGOmGly9zagaXKXtfmZzi+VCvgo=
+podPriorityClassName: staging
```

Comment:
```suggestion
podPriorityClassName: staging
```

Reply:
Ah sorry I just saw that there was no new line in the end so tried to fix - didn't work. don't worry, no problem with the line endings, just looked weird in github diff